apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/wpcom-nux.js

```diff
@@ -125,6 +125,7 @@ function getWpcomNuxPages() {
 			imgSrc: previewImage,
 			alignBottom: true,
 		},
+		// Last step
 		{
 			heading: __( 'Hidden until you’re ready', 'full-site-editing' ),
```

Comment:
i18n comments should be located immediately before the targeted string. cc: @Automattic/i18n to confirm
Also, it should probably be a little more explicit.

```suggestion
			heading: `/* Last step of the Welcome modal displayed when first using the editor */ __( 'Hidden until you’re ready', 'full-site-editing' ),
```